style.css

```diff
@@ -709,24 +769,1211 @@ body {
+    font-family: "Poppins", sans-serif;
+    padding: 2px 0 0 10px;
+    color: #fff;
+  }
+
+  .cjs {
+    width: 92px;
+    height: 28px;
+    background: #ebebff;
+    border-radius: 8px;
+    transition: transform 1s, box-shadow 0.5s;
+    box-shadow: 0 0 0 rgba(0, 0, 0, 0.3);
+  }
+
+  .cjs:hover {
+    transform: scale(1.2) rotate(360deg);
+    box-shadow: 10px 10px 30px rgba(0, 0, 0, 0.3);
+    background: #172b4d;
+  }
+
+  li.cjs:hover {
+    font-size: 14px;
+    font-weight: 500;
+    font-family: "Poppins", sans-serif;
+    padding: 5px 0 0 8px;
+    color: #fff;
+  }
+
+  .cseepro {
+    margin-left: 5%;
+    margin-top: -10%;
+    width: 15%;
+    height: 50%;
+    cursor: pointer;
+    border: 1px solid;
+    border-radius: 10px;
+    padding: 10px;
+    color: #6070ff;
+    background-color: white;
+    font-size: 18px;
+  }
+
+  .cseepro:hover {
+    background-color: #6070ff;
+    color: white;
+    border-color: #6070ff;
+  }
+
+  .cseepro:active {
+    background-color: #2230d2;
+    color: #fff;
+    border-color: #2230d2;
+  }
+
+  .about {
+    display: flex;
+    width: 100%;
+    height: 600px;
+    background: #fff;
+    border-top-right-radius: 100px;
+  }
+
+  .about_me {
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+  }
+
+  .about_me h2 {
+    height: 150px;
+    margin-top: 80px;
+    margin-left: 10%;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .about_me p {
+    height: 146px;
+    width: 45%;
+    margin-left: 10%;
+    margin-top: -75px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .about_me h3 {
+    height: 24px;
+    margin-left: 10%;
+    margin-top: 50px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .socialm {
+    display: flex;
+    width: 10%;
+    height: 3%;
+    margin-left: -54%;
+    gap: 45px;
+    margin-top: 385px;
+    list-style-type: none;
+  }
+
+  .language {
+    display: flex;
+    width: 40%;
+    flex-direction: column;
+    margin-top: 210px;
+    margin-left: auto;
+  }
+
+  .language h1 {
+    height: 24px;
+    margin-top: -75%;
+    margin-left: -22%;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    margin-right: 0.5em;
+  }
+
+  .arrow {
+    height: 3%;
+    width: 6%;
+    margin-left: 50%;
+    margin-top: -30px;
+  }
+
+  .lang_drop {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    list-style-type: none;
+    margin-top: 40px;
+    gap: 60px;
+  }
+
+  .javas {
+    width: 20%;
+    height: 100%;
+    margin-left: -45px;
+    border-radius: 8px;
+    background: #ebebff;
+  }
```

Comment:
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but please make sure you're properly following the given Figma design as the background color of this selector should be `#F7F7F9` and not `#ebebff`